vtr_flow/scripts/python_libs/vtr/flow.py

```diff
@@ -0,0 +1,321 @@
+import sys
```

Comment:
This is just for the previously mentioned parse functionality of which I am not yet working on.